extensions/schema-compare/src/dialogs/schemaCompareOptionsDialog.ts

```diff
@@ -95,14 +100,12 @@ export class SchemaCompareOptionsDialog {
 	}
 
 	private async reset(): Promise<void> {
+		this.resetClicked = true;
 		let service = (vscode.extensions.getExtension(mssql.extension.name).exports as mssql.IExtension).schemaCompare;
 		let result = await service.schemaCompareGetDefaultOptions();
```

Comment:
As with the other - do we really need to get the default options again? You already have the original deploymentOption values in the model so couldn't you just reset the lookup map and be done?

(which is currently being done in the execute click which doesn't seem correct)